packages/create-svelte/templates/default/package.json

```diff
@@ -13,7 +13,7 @@
 		"svelte": "^3.48.0",
 		"svelte-preprocess": "^4.10.6",
 		"typescript": "^4.8.2",
-		"vite": "3.1.0-beta.2"
```

Comment:
we should probably add `create-svelte` to the changelog as well

Reply:
This isn't the package.json that's used when people run the CLI. That's package.template.json, which was already updated.